runtime/doc/lua.txt

```diff
@@ -3023,6 +3023,11 @@ vim.fs.find({names}, {opts})                                   *vim.fs.find()*
                  • {limit}? (`number`, default: `1`) Stop the search after
                    finding this many matches. Use `math.huge` to place no
                    limit on the number of matches.
+                 • {skip}? (`(fun(dir_name: string): boolean)?`) Do not
+                   traverse matching directories. If omitted, all directories
+                   are searched recursively.
+                 • {follow}? (`boolean`, default: `true`) Follow symbolic
```

Comment:
This could be worth adding, since checking for a symlink may be more than a simple matter of a single `fs_stat` ?

Reply:
it does probably need extra checks in windows, since I see the test failed there

Reply:
TIL about `getftype()`, maybe that works correctly. Might be worth testing it.